data/brands/shop/bakery.json

```diff
@@ -1042,6 +1042,19 @@
         "shop": "bakery"
       }
     },
+    {
+      "displayName": "Kuhn",
+      "id": "kuhn-6950bd",
+      "locationSet": {"include": ["ch"]},
+      "tags": {
+        "bakehouse": "no",
```

Comment:
https://wiki.openstreetmap.org/wiki/Key:bakehouse

According to OSM Wiki……If it is not a bakery but it still be tagged as bakery, this will be strange.

Could you tell us more about why add this tag?

Reply:
Removed it.